tests/cucumber/features/channels/channel_creation.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Channel creation
+  As an authenticated user
+  I want to create a channel
+
+  Background:
+    Given I am on the home page
+    And I am signed in
+    When I click on the new channels button
```

Comment:
Could you just have this say "Given I am on the new channels page"? That seems like it would kill two birds with one stone.